src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation.md

```diff
@@ -30,6 +34,21 @@ This page gives all the necessary steps to install the Retention Management addo
 
 ## Configuration
 
+### Standard Mode
+
+#### {{> anchor 'standard-amazon-s3'}} Amazon S3
+
+The default configuration consists on using the same S3 bucket for records than for the standard documents.
```

Comment:
Speaking about that, it is not good advice to use the same bucket for records and regular blobs. It led to situations such as https://jira.nuxeo.com/browse/NXP-30739
Ideally, records should be stored in a dedicated bucket where s3 Object Lock feature is enabled.
If sharing the same bucket, the underlying blob providers should at least use have different bucket prefix.
We should review the config given in https://doc.nuxeo.com/nxdoc/nuxeo-retention-installation/#configure-the-records-bucket-in-amazon-s3-online-storage accordingly which I can do in the scope of https://jira.nuxeo.com/browse/NXP-30640

Reply:
my understanding is that if you use the "legacy" binary manager, there is no impact on the storage side to move a document to a record (no move, no renaming,...), so I don't see the impact on the GC side.
The issues are only if you use the new blob provider.
Am I correct ?
Thanks